webui/react/src/pages/Cluster.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useCallback } from 'react';
+
+import Grid from 'components/Grid';
+import Page from 'components/Page';
+import ResourceChart from 'components/ResourceChart';
+import Spinner from 'components/Spinner';
+import AgentsCtx from 'contexts/Agents';
+import emptyMessage from 'styles/emptyMessage.module.scss';
+import { Resource } from 'types';
+import { categorize } from 'utils/data';
+
+import css from './Cluster.module.scss';
+
+const Cluster: React.FC = () => {
+  const agents = AgentsCtx.useStateContext();
+
+  const getResources = useCallback(() => {
+    if (!agents.data) return {};
+    const resourceList = agents.data?.map(a => a.resources).reduce((a, c) => {
+      a.push(...c); return a;
+    });
```

Comment:
nit: what do would you say to these?
- avoid one letter variable names.
- split chained links into their own lines if there are more than 1
```
const resourceList = agents.data?
  .map(agent => agent.resources)
  .reduce((acc, resources) => {
    acc.push(...resources);
    return acc;
  }, []);
```